src/openforms/js/components/admin/form_design/logic/ActionSet.js

```diff
@@ -0,0 +1,177 @@
+import React, {useEffect} from 'react';
+import {Action} from './Action';
+import {useImmerReducer} from 'use-immer';
+import usePrevious from 'react-use/esm/usePrevious';
+import isEqual from 'lodash/isEqual';
+import PropTypes from "prop-types";
+import Trigger from "./Trigger";
```

Comment:
I wish we had `isort` for JS

Please group third-party dependencies at the top, follow by a blank line and then own-component imports.